components/prism-ichigojam.js

```diff
@@ -0,0 +1,15 @@
+// according to the offical reference https://ichigojam.net/IchigoJam-BASIC-reference.pdf
```

Comment:
Is there an english version of this reference? If there is we should add it to this comment. If there is not, could you just add `(JP)` somewhere in the comment?